packages/orchestration/README.md

```diff
@@ -23,6 +23,7 @@ This package incorporates generative AI orchestration capabilities into your AI
   - [Custom Request Configuration](#custom-request-configuration)
   - [Custom Destination](#custom-destination)
 - [Local Testing](#local-testing)
+- [Error Handling](#error-handling-1)
```

Comment:
```suggestion
- [Error Handling](#error-handling)
```

Reply:
This is added automatically by my vscode extension and somehow vscode navigates correctly..